.gitignore

```diff
@@ -18,6 +18,7 @@
 /Temp
 ExportedObj/
 Temp/
+/Assets/LeapMotionModules/Hands/Models/TestModels/
```

Comment:
You can put a gitignore inside the folder that contains the file you want to ignore, that way you don't need to hard-code paths.